libs/sdk-core/src/models.rs

```diff
@@ -629,6 +629,11 @@ pub struct NodeState {
     pub max_single_payment_amount_msat: u64,
     pub max_chan_reserve_msats: u64,
     pub connected_peers: Vec<String>,
+
+    /// Maximum receivable in a single payment (single channel)
```

Comment:
I know this line was suggested by @ok300, but a little nit here still.

```suggestion
    /// Maximum receivable in a single payment without requiring a new channel open.
```